Main.css

```diff
@@ -0,0 +1,412 @@
+    color: white;
+    font: bold 18px/18px "Roboto Condensed",sans-serif;
+}
+.sign-up-btn::after{
+    content: "";
+display: inline-block;
+width: 8px;
+height: 14px;
+margin-left: 10px;
+background: url(https://g.newsweek.com/www/images/icon-arrow-right.svg) center/contain no-repeat;
+}
+.nav-menu-border{
+    border-right: 1px solid #d9d9d9;
+    padding-right: 10px;
+    margin-right: 10px;
+    
+}
+.nav{
+    border-bottom-color: #f4f5f6;
+    padding-top: 15px;
+padding-bottom: 15px;
+border-bottom-width: 2px;
+border-bottom-style: solid;
+z-index: 1;
+}
+.flex{
+    display: flex;
+    justify-content: center;
+
+}
+.flex-left{
+    display: flex;
+    justify-content: start;
+
+}
+.bold{
+    font-weight: bold;
+}
+#footer-logo{
+    color: #f72210;
+    font-size:300%;
+font-weight: bold;
+line-height: 1.5;
+font-family: "Roboto Condensed",sans-serif;
+margin-right: 15px;
+}
+.logo-container{
+    align-items: baseline;
+}
+footer a:hover{
+    text-decoration: none;
+    color: #666;
+}
+.social-links{
+    display: flex;
+    justify-content: end;
+   
+
+}
+.social-links i{
+   font-size: 26px;
+   padding: 8px;
+}
+.social-container{
+    display: flex;
+    justify-content: end;
+    align-self: flex-end;
+}
+.block-title{
+    color: red;
+    padding: 10px;
+}
+
+.navbar-toggle-icon span{
+    background-color: grey;
+    display: inline-block;
+    height: 3px;
+    width: 20px;
+}
+.navbar-dropdown{
+    position:absolute;
+    right: 0;
+    left: auto;
+    background-color: white;
+    list-style: none;
+    margin-top: 8px;
+    padding-left: 0;
+    font-weight: bold;
+}
+.navbar-dropdown li:hover{
+    color: white;
+    background-color: black;
+}
+.white-text{
+    color: white;
+}
+span.white-text{
+    font-family: 'Merriweather', serif;
+    font-weight: bold;
+    padding: 6px;
+    padding-right: 10px;
+}
+.red-text{
+    color: red;
+    
+}
+span.red-text{
+    font-family: 'Merriweather', serif;
+    font-weight: bold;
+    padding: 6px;
+    padding-left: 10px;
+    margin-bottom: 0;
+}
+
+    .unstyled-hyperlink:hover{
+        text-decoration: none;
+    }
+    .hidden{
+       color:transparent;
+    }
+    .more-stories{
+        overflow: hidden;
+    }
+    date{
+        font: 16px/34px "Roboto Condensed",sans-serif;
+    }
+    .no-wrap{
+        white-space: nowrap;
+        overflow: hidden;
+    }
+    .float-left{
+        float: left;
+    }
+    .float-right{
+        float: right;
+    }
+    @media only screen and (min-width: 945px){
+      
+        .navbar-toggler{
+            display: none;
+        }
+    }
+@media only screen and (max-width: 1280px){
+    #static-nav{
+        display: none;
+    }
+    .navbar-toggler{
+        display: inline-block;
+    }
+}
+@media only screen and (max-width: 945px){
+    .hide-small{
+        display: none;
+    }
+    .newsweek-logo{
+        font-size: 30px;
+    }
+}
+@media only screen and (min-width: 768px){
+      
```

Comment:
### `HINT`

Consider adding a smaller font size for the navigation bar for smaller screens. Also, the footer can use some repositioning for smaller screens. See illustration below:

| Current | Preferred |
| :-----: | :------: |
| ![bootstrap footer current](https://user-images.githubusercontent.com/36057474/62749163-47baff00-ba53-11e9-8352-3ff6abb578c0.png) | ![bootstrap footer preffered](https://user-images.githubusercontent.com/36057474/62749162-47226880-ba53-11e9-9015-a50c90016558.png) |